app/src/main/java/com/zeapo/pwdstore/crypto/PgpActivity.kt

```diff
@@ -161,7 +161,11 @@ class PgpActivity : AppCompatActivity(), OpenPgpServiceConnection.OnBound {
 
                 title = getString(R.string.new_password_title)
                 crypto_password_category.apply {
-                    setText(getRelativePath(fullPath, repoPath))
+                    val path = getRelativePath(fullPath, repoPath)
+                    if (path.isEmpty())
```

Comment:
We might want to move this below the Autofill checks that make this field editable and gate hiding it on `isEnabled != true`. Otherwise, users would be unable to change this from an empty path if Autofill should ever suggest one (I don't think it ever does though, it always suggests at least a `/`).